pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -69,11 +69,12 @@ type CronWorkflowSpec struct {
 	When string `json:"when,omitempty" protobuf:"bytes,12,opt,name=when"`
 }
 
-// v3.6 and after: StopStrategy defines if the CronWorkflow should stop scheduling based on a condition
+// StopStrategy defines if the CronWorkflow should stop scheduling based on an expression. v3.6 and after
```

Comment:
The version annotation being consistently the first thing is intentional so that it is the first thing you see. This change makes this inconsistent with every other version annotation in the codebase, like lines 64-68 above.

I believe you made this change for GoDoc purposes, which does raise a conundrum since that apparently _requires_ it to start the same way? If it's just a convention, the consistency and visibility should superseded that.

Reply:
I did that to make it GoDoc compliant.

Reply:
That does not address my comment and, as I wrote above, does not match the entirety of rest of the code however...